app/views/customers/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Customer Name: <%= @customer.name %></h1>
+<section id="customer_items">
+<h2>Items</h2>
+<% @customer.items.each do |item| %>
+  <section id= "item-<%= item.id %>">
+  <p><%= "Item: #{item.name}" %></p>
+  <p><%= "Price: $#{item.price}" %></p>
+  <p><%= "Supermarket Name: #{item.supermarket.name}" %></p>
+  </section> 
```

Comment:
Same here, the code between this section block should be indented, so lines 6-7 should be indented ones. We want to show the html structure with our indentations.